src/librustdoc/html/render.rs

```diff
@@ -793,6 +793,8 @@ fn write_shared(
           static_files::BRUSH_SVG)?;
     write(cx.dst.join(&format!("wheel{}.svg", cx.shared.resource_suffix)),
           static_files::WHEEL_SVG)?;
+    write(cx.dst.join(&format!("down-arrow{}.svg", cx.shared.resource_suffix)),
```

Comment:
Whenever this gets merged, we'll need to update docs.rs to make it start caching the new SVG. (I'm remembering this detail late for a few other open PRs as well.)